components/UploadForm.tsx

```diff
@@ -59,38 +42,22 @@ const FormUpload: React.FC<Props> = (props) => {
           />
         </FormControl>
         <FormControl isRequired>
-          <FormLabel htmlFor="category">
-            {t('components.uploadForm.category.label')}
-          </FormLabel>
-          {isCustom ? (
-            <Input
-              id="category"
-              placeholder={t('components.uploadForm.category.placeholder')}
-              isRequired
-              onChange={(e) => setCategory(e.currentTarget.value)}
-            />
-          ) : (
-            <Select
-              id="category"
-              placeholder={t('components.uploadForm.category.default')}
-              value={category}
-              isRequired
-              onChange={(e) => setCategory(e.currentTarget.value)}
-            >
-              <option value="option1">Option 1</option>
-              <option value="option2">Option 2</option>
-              <option value="option3">Option 3</option>
-              <option value="custom">{t('common.other')}</option>
-            </Select>
-          )}
-        </FormControl>
-        <FormControl isRequired>
-          <FormLabel>{t('components.uploadForm.file.label')}</FormLabel>
           <Input
             onChange={(e) => setFiles(e.currentTarget.files)}
             isRequired
             type="file"
             accept="image/*,.pdf"
+            ref={inputRef}
+            style={{ display: 'none' }}
+          />
+          <FormLabel>{t('components.uploadForm.file.label')}</FormLabel>
+          <Input
+            onClick={() => inputRef?.current?.click()}
+            value={
+              files?.item(0)?.name ||
+              t('components.uploadForm.file.placeholder').toString()
```

Comment:
`.toString()` here is not necessary

Reply:
the linter is ngamuk when I didn't put that, do you have any suggestion

Reply:
@adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya